apps/src/netsim/NetSimApi.js

```diff
@@ -0,0 +1,65 @@
+/**
+ * @overview Wraps NetSim REST API by extending our standard client API.
+ * @see clientApi.js
+ * @see net_sim_api.rb
+ */
+/* jshint
+ funcscope: true,
+ newcap: true,
+ nonew: true,
+ shadow: false,
+
+ maxlen: 90,
+ maxparams: 3,
+ maxstatements: 200
+ */
+/* global $ */
+'use strict';
+
+
+var clientApi = require('@cdo/shared/clientApi');
+
+/**
+ * @name NetSimApi
+ * @extends ClientApi
+ */
+var newApi = {
```

Comment:
Rather than extending the ClientApi, suggest we just define a distinct API.  (They might change the ClientAPI and we wouldn't want to track their changes.)